cmake/kokkos_arch.cmake

```diff
@@ -845,14 +845,35 @@ IF (KOKKOS_ENABLE_OPENACC)
       Clang -Xopenmp-target=nvptx64-nvidia-cuda -march=${CLANG_CUDA_ARCH}
             -fopenmp-targets=nvptx64-nvidia-cuda
     )
+    IF(DEFINED ENV{CUDA_PATH})
+      COMPILER_SPECIFIC_LINK_OPTIONS(Clang -L$ENV{CUDA_PATH}/lib64)
+    ENDIF()
+    COMPILER_SPECIFIC_LIBS(
+      Clang -lcudart
+      NVHPC -cuda
+    )
   ELSEIF(KOKKOS_AMDGPU_ARCH_FLAG)
     COMPILER_SPECIFIC_FLAGS(
       Clang -Xopenmp-target=amdgcn-amd-amdhsa -march=${KOKKOS_AMDGPU_ARCH_FLAG}
             -fopenmp-targets=amdgcn-amd-amdhsa
     )
+    IF(DEFINED ENV{ROCM_PATH})
+      COMPILER_SPECIFIC_FLAGS(Clang -I$ENV{ROCM_PATH}/include)
+      COMPILER_SPECIFIC_LINK_OPTIONS(Clang -L$ENV{ROCM_PATH}/lib)
+    ENDIF()
+    COMPILER_SPECIFIC_LIBS(Clang -lamdhip64)
```

Comment:
Is finding `ROCm` not sufficient?

Reply:
As I understand, the ROCM module is loaded only for the HIP backend, but I don't know how to set cmake configurations so that it is also loaded when the OpenACC backend targets AMD GPUs; can you (or anyone) help on this?

Reply:
I think you would need to configure with -DKokkos_ENABLE_ROCM=ON to tell Kokkos to find it

Reply:
Just adding `Kokkos_ENABLE_ROCM=ON` does not seem to work. (cmake/Modules/FindTPLROCM.cmake does not set include path.)

Reply:
You would still have to register that your target depend on it.

Reply:
How to register it? (FindTPLROCM.cmake also needs to be updated to include the path for the hip runtime header.)